modules/integration_azure-container-apps/conf/00-heartbeat.yaml

```
@@ -0,0 +1,16 @@
+module: Azure Container Apps
+name: heartbeat
+filtering: "filter('resource_type', 'Microsoft.App/containerApps') and filter('aggregation_type', 'minimum')"
+
+transformation: true
+aggregation: ".mean(by=['azure_resource_id'])"
+
+signals:
+  signal:
+    metric: "WorkingSetBytes"
+
+rules:
+  critical:
+    threshold: 0
+    comparator: "="
+    lasting_duration: 5m
```

No condition needed for heartbeat.
```suggestion
rules:
  critical:
```